docs/docs/rad-lab-ui/ui_configure/ui-meta.md

```diff
@@ -61,10 +67,56 @@ variable "boot_disk_size_gb" {
 ```
 
 ## Updatesafe
+
 After a module has been deployed (the project created and resources created within it), some Terraform updates can be **destructive**. To warn the users that they may be performing a potentially destructive action, the `updatesafe` key word is added (or omitted) to the `UIMeta`.
 
 Any time a user makes an update to an existing deployment, the `updatesafe` values are checked. If **ANY** of the changed variables do **NOT** have the `updatesafe` keyword, the user will be warned that they likely will lose data by applying the update. Put another way, only if **ALL** the changed variables are `updatesafe` will the prompt be skipped.
 
+## Mandatory
+
+Some of the variables are required to have value for deployment, in order to make it mandatory of the variables is determined by the `mandatory` keyword. User must have to enter some values for it.
+
+**(Example)** `billing_account_id` will be shown as mandatory for users and User must have to provide the input.
+
+```terraform
+variable "billing_account_id" {
+  description = "Billing Account associated to the GCP Resources. {{UIMeta group=0 order=3 mandatory }}"
+  type        = string
+}
+```
+
+## Depends On
+
+Some of the variables are depends on other variables input, in order to make it dependent of the variables is determined by the `dependson` keyword. So based on user input and depends on condition satified to be shown the dependent variables.
+
+**(Example)** `container_image_repository` will be shown once `create_container_image` value will be true, similarly `ip_cidr_range` will be shown once `create_network` and `create_usermanaged_notebook` value will be true , similarly for other variables it will work based on `dependson` conditions with `||` and `&&` operands.
+
+```terraform
+variable "container_image_repository" {
+  description = "Container Image Repo, only set if creating container image notebook instance by setting \`create_container_image\` variable to true. {{UIMeta group=2 order=4 dependson=(create_container_image==true) mandatory }}"
+  type        = string
+  default     = ""
+}
+
+variable "ip_cidr_range" {
+  description = "Unique IP CIDR Range for AI Notebooks subnet {{UIMeta group=3 order=5 dependson=(create_network==true&&create_usermanaged_notebook==true) mandatory}}"
+  type        = string
+  default     = "10.142.190.0/24"
+}
+
+variable "network_name" {
+  description = "Name of the network to be created. {{UIMeta group=3 order=2 dependson=(create_usermanaged_notebook==true||enable_gpu_driver==true) mandatory}}"
+  type        = string
+  default     = "ai-notebook"
+}
+
+variable "subnet_name" {
+  description = "Name of the subnet where to deploy the Notebooks. {{UIMeta group=3 dependson=((enable_gpu_driver==true||create_usermanaged_notebook==true)&&(create_network==true||set_external_ip_policy==true)) mandatory}}"
+  type        = string
+  default     = "subnet-ai-notebook"
+}
```

Comment:
Please remove the **first** and **last** parentheses wrapping the `dependsons` content